src/index.html

```diff
@@ -1,13 +1,433 @@
+                <div class="grid">
+                  <div class="grid__item
+                              grid__item--desktop-1-4 
+                              grid__item--iPad-1-2
+                  ">
+                    <div class="bike__photo-container">
+                      <img 
+                        src="images/photos/row_3.png" 
+                        alt="AutoUnlock" 
+                        class="bike__photo"
+                      >
+                    </div>
+                  </div>
+                  <div class="grid__item 
+                              grid__item--iPad-3-6
+                              grid__item--desktop-5-12
+                  ">
+                    <div class="bike__photo-container">
+                      <img 
+                        src="images/photos/row_3.2.png" 
+                        alt="AutoUnlock" 
+                        class="bike__photo"
+                      >
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="bike__description">
+                <div class="bike__title">Auto Unlock</div>
+                The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+              </div>
+            </div>
+            <div class="grid grid--desktop">
+              <div class="grid__item
+                          grid__item--iPad-2-5
+                          grid__item--desktop-6-7
+              ">
+                <a href="#" class="button">
+                  Explore
+                </a>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="page__section contact-us" id="contacts">
+          <div class="container">
+            <h2 class="page__section-title">
+              Contact us
+            </h2>
+
+            <div class="grid grid--desktop">
+              <div class="grid__item
+                          grid__item--iPad-1-3
+                          grid__item--desktop-1-5"
+              >
+                <form action="#">
+                  <input 
+                    name="name" 
+                    type="text"
+                    placeholder="Name"
+                    class="contact-us__field form-field"
+                    required
+                  >
+                  <input 
+                    name="email"
+                    placeholder="Email"
+                    type="email"
+                    class="contact-us__field form-field"
+                    required
+                  >
+                  <textarea 
+                    name="message"
+                    placeholder="Message"
+                    class="
+                      contact-us__field
+                      contact-us__field--last
+                      form-field 
+                      form-field--textarea"
+                  >
+                  </textarea>
```

Comment:
```suggestion
                  <textarea 
                    name="message"
                    placeholder="Message"
                    class="
                      contact-us__field
                      contact-us__field--last
                      form-field 
                      form-field--textarea"
                  ></textarea>
```

It should fix extra spaces in textarea field